### ISSUE TITLE
Periodic targets IPTT: Always show targets even when there are no results yet

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#375]
## This is an issue we began to tackle towards the end of Winterhawks, that turned out to be a _high risk for serious regressions_, with unexpected results showing up in the IPTTs. (If I recall correctly, calculations were unexpectedly being run multiple times, with results displayed that were multiples of the result you'd expect.) 

Due to the regression risk and complexity of the queries involved, I propose that we wait until IPTT-related unit tests are in place before tackling this ticket -- i.e. ticket #468.

# Issue
Just as we always show targets on the Indicators page even before results are added, we always want to display targets in the IPTT report. Currently, the target columns are empty until at least one result is reported.

## Housekeeping (where applicable)
- [ ] Unit test/s written 
- [ ] i18n strings included
- [ ] Is accessible to people using screen readers

**Comment by jennyhavah** [comment]
Validated.

**Comment by jennyhavah** [comment]
We discovered a bug that we think was related to this fix, that caused IPTT calculations to be wrong. (See #412)

We need to find a different method of showing targets when there are no results, that doesn't negatively affect calculations.

**Comment by camgnostic** [comment]
This is addressed in branch `iptt-refactor-3` merging into Satsuma later this week.

**Comment by jennyhavah** [comment]
PASSED.

I tested this ticket using the QA Program "No Results Here". For each target frequency, I looked to see if targets were displayed even though there are no results reported -- both for LoP and individual target periods.

- [x] LoP only -- for now, will retest when #1606 is done.
- [x] Midline/Endline
- [x] Annual
- [x] Semi-annual
- [x] Tri-annual
- [x] Quarterly
- [x] Monthly
- [x] Events

I also checked out a number of real programs and consistently saw targets displayed even when no results were reported -- either for LoP or a target period.
